docs/advanced-topics/advanced-features.rst

```diff
@@ -311,14 +311,22 @@ primary descriptor is in the root directory of its git repository.
 WDL Launcher Configuration
 --------------------------
 
-By default, WDL tools/workflows will automatically be run with
-`cromwell <https://github.com/broadinstitute/cromwell>`__ 36.
+By default, WDL tools/workflows will automatically be run by the Dockstore CLI
+with the `Cromwell <https://github.com/broadinstitute/cromwell>`__ version listed below.
+
++-------------+-----------------------+
+| CLI version | Cromwell version used |
++=============+=======================+
+|     1.8.0   |          44           |
```

Comment:
Perhaps 1.8 or 1.8.x? Has it changed in the 1.8 timeframe? If not, I lean towards 1.8.